aws-platform-ui-main/vars.tf

```diff
@@ -163,3 +163,19 @@ variable "snapshot_frequency_hours" {
   type        = number
   default     = 2
 }
+
+variable "coredns_rewrite_rules" {
+  type = list(object({
+    query  = string
+    target = string
+  }))
+  default = [
+    { query = "orderer0.luther.systems", target = "fabric-orderer0.fabric-orderer.svc.cluster.local" },
+    { query = "orderer1.luther.systems", target = "fabric-orderer1.fabric-orderer.svc.cluster.local" },
+    { query = "orderer2.luther.systems", target = "fabric-orderer2.fabric-orderer.svc.cluster.local" },
+    { query = "peer0.org1.luther.systems", target = "fabric-peer0-org1.fabric-org1.svc.cluster.local" },
+    { query = "peer1.org1.luther.systems", target = "fabric-peer1-org1.svc.cluster.local" },
```

Comment:
`fabric-peer1-org1.svc.cluster.local` seems to be an outlier here. Should it include `.fabric-org1`?

Reply:
good catch that looks like a mistake, will fix